File: TodoList/TodoList/domain/entity/CardAction.swift
//
//  CardAction.swift
//  TodoList
//
//  Created by Bibi on 2022/04/12.
//

import Foundation

enum CardAction {
    case add
    case remove
    case update
    case move
    
    var name: String {
        switch self {
        case .add:
            return "등록"
        case .remove:
            return "삭제"
        case .update:
            return "변경"
        case .move:
            return "이동"
        }
    }
}

File: backend/src/main/java/com/team15/todoapi/domain/CardAction.java
package com.team15.todoapi.domain;

public enum CardAction {
	ADD(1),
	REMOVE(2),
	UPDATE(3),
	MOVE(4);

	private final int code;

	CardAction(int code) {
		this.code = code;
	}

	public int getCode() {
		return code;
	}
}
